sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/SparkMetadataOperationSuite.scala

```diff
@@ -28,23 +30,40 @@ class SparkMetadataOperationSuite extends HiveThriftJdbcTest {
 
   test("Spark's own GetSchemasOperation(SparkGetSchemasOperation)") {
```

Comment:
Does this operation work for permanent views?

Reply:
this is DB layer API, we need to fix other type of operation too